card/src/main/java/com/adyen/checkout/card/StoredCardDelegate.kt

```diff
@@ -88,6 +88,14 @@ class StoredCardDelegate(
         return SocialSecurityNumberUtils.validateSocialSecurityNumber(socialSecurityNumber)
     }
 
+    override fun validateKcpBirthDateOrTaxNumber(kcpBirthDateOrTaxNumber: String): FieldState<String> {
+        return CardValidationUtils.validateKcpBirthDateOrTaxNumber(kcpBirthDateOrTaxNumber)
```

Comment:
Do we show the KCP fields in the stored component? If not then this should always return Valid